ci/provisioning/post_provision_config_nodes.sh

```diff
@@ -107,6 +107,10 @@ rm -f  /etc/yum/vars/releasever
 # and catted to the remote node along with this script
 post_provision_config_nodes
 
+if lspci | grep -i nvme; then
+  daos_server storage prepare -n --reset && rmmod vfio_pci && modprobe vfio_pci
```

Comment:
probably want to add a comment that references the SPDK issue that this addresses to help people understand why this is needed

Reply:
Is this not just some temporary debugging to gather some information, etc.?  Is it expected that whatever adding this here fixes can and will be seen in the field in production installations?  If so, this is the wrong place for this then.  If this can happen in the wild, it needs to be added into DAOS proper, not the test-node setup script.

Reply:
Fixes to the test harness are being done by Phil in https://daosio.atlassian.net/browse/DCO-10397.
DAOS issues will be handled by other tickets.

Anything other than what is needed CORCI-1200 will eventually be removed from this PR.  Right now it is just convenient to use for multiple debugging purposes.